docs/content/policy-reference.md

```diff
@@ -793,6 +793,7 @@ Note that the opa executable will need access to the timezone files in the envir
 | Built-in | Description | Wasm Support |
 | ------- |-------------|---------------|
 | <span class="opa-keep-it-together">``output := crypto.x509.parse_certificates(certs)``</span> | ``certs`` is base64 encoded DER or PEM data containing one or more certificates or a PEM string of one or more certificates. ``output`` is an array of X.509 certificates represented as JSON objects. | ``SDK-dependent`` |
+| <span class="opa-keep-it-together">``output := crypto.x509.parse_and_verify_certificates(certs)``</span> | ``certs`` is base64 encoded DER or PEM data containing two or more certificates where the first is a root CA, the last is a leaf certificate, and all others are intermediate CAs. ``output`` is of the form ``[valid, certs]``. If the input certificate chain could be verified then ``valid`` is ``true`` and ``certs`` is an array of X.509 certificates represented as JSON objects. If the input certificate chain could not be verified then ``valid`` is ``false`` and ``certs`` is ``[]``. | ``SDK-dependent`` |
```

Comment:
🤔 Is this in line with what comparable builtins do? I'd have expected `[valid, certs]` or `undefined` (i.e. returning some builtin-error type that would end up in an evaluation to `undefined` if not using strict builtin errors).

Reply:
Are you referring to the choice of returning an empty array if `valid` is false? If so, then yes this is in line with what JWT `decode_verify` returns. If someone is explicitly checking the cert chain, I think it makes sense to not return the certs if the verification fails to make sure the non-verified certs aren't accidentally trusted in the policy.

Reply:
I'm thinking about this more broadly -- why return anything if `valid` is false? We could return the `certs` array if it's valid, and not return anything at all if not.

But I've been scanning the implementation of other ` io.jwt.*` functions (like `io.jwt.verify_rs256`), and they do return true or false (not true or undefined). So, yeah, let's keep this as you've implemented it, it should be the choice more consistent with the rest 👍